vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ky:govstatus"
+
+
+def _get_id(site: dict) -> str:
+    return site["id"]
+
+
+def _get_name(site: dict) -> str:
+    return site["name"]
+
+
+def _get_city(site: dict) -> str:
+    return site["address"]["city"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["address"]["street1"],
+        city=_get_city(site),
+        zip=site["address"]["zip"],
+        state=site["address"]["state"],
+    )
+
+
+def _get_location(site: dict):
+    latitude = site["lat"]
+    longitude = site["long"]
+    if latitude == "" or longitude == "":
+        return None
+    return schema.LatLng(
+        latitude=float(latitude),
+        longitude=float(longitude),
+    )
+
+
+def _get_contacts(site: dict):
+    ret = []
+
+    if "register_phone" in site:
+        raw_phone = site["register_phone"]
+        if raw_phone != "":
+            raw_phone = raw_phone.lstrip("tel:")
```

Comment:
I haven't looked deeply into this myself, but have you looked at using a pre-built phone number parser for this like https://github.com/daviddrysdale/python-phonenumbers

Reply:
Ooh, looks neat! Has some bad behavior on this set, so maybe not worth moving over this validator specifically:
```
>>> import phonenumbers
>>> print(phonenumbers.parse("tel:270-685-7100 option #6", "US"))
Country Code: 1 National Number: 2706857100678466 Extension: 6
```

Will keep in mind for future cases, though.